api-service/crowdsource-webapp/src/app/app.component.spec.ts

```diff
@@ -14,6 +14,7 @@ describe('AppComponent', () => {
     }).compileComponents();
   }));
 
+  /*
   it('should create the app', () => {
```

Comment:
If this is a set you want to come back to, better to use `xit` instead of commenting it out. If you don't want it, you can delete it. If you leave comment, or make it an xit test, add a comment to explain a little.

Reply:
Oops, I actually meant to fix these and forgot about them. Thanks for catching.